lib/site_prism/page.rb

```diff
@@ -152,11 +154,13 @@ def load_html_string(string)
       yield self if block_given?
     end
 
-    def load_html_website(html, &block)
+    def load_html_website(html, with_validations, &block)
       expanded_url = url(html)
       raise SitePrism::NoUrlForPageError unless expanded_url
 
       visit expanded_url
+      return yield self unless with_validations
```

Comment:
This will cause issues as a different yield will be returned dependent on situation. The return value of this method will be yielded later on in the method invocation chain.